jaseci_kit/jaseci_kit/modules/object_detection/api.py

```diff
@@ -0,0 +1,140 @@
+from fastapi import FastAPI, Form, File, UploadFile
```

Comment:
Can you separate the yolo model into its own PR?

Reply:
I was trying this but it kept updating the previous PR and if I had to create a different branch the previous PR is still attached which causes problems

Reply:
Make a new branch for yolov5 and open a new PR.